/*public class Exception {
    public static void main(String args[]){
        int a=10,b=0;
        int c=a/b;
        System.out.println("result= "+c);
    }
}*/


/*class Exception{
    public static void main(String args[]){
        int a=10,b=0;
        try {
            int c=a/b;
            System.out.println("result: "+c);
        } catch (ArithmeticException e) {
            System.out.println("Division by zero is not allowed");
        }
        finally{
            System.out.println("Program has been executed!");
        }
    }
}*/